schemas/macro_service.schema.json

```diff
@@ -139,6 +157,10 @@
     "Response": {
       "$description": "A response to a [MacroRequest] or [HostRequest].",
       "properties": {
+        "requestId": {
```

Comment:
FWIW I would also be okay with just "id", but possibly this is a bit clearer, up to you :)

Reply:
I think this is better - harder to make a mistake when creating a response object